l10n/configs/pontoon.toml

```diff
@@ -164,6 +164,25 @@ locales = [
 [[paths]]
     reference = "en/mozorg/mission.ftl"
     l10n = "{locale}/mozorg/mission.ftl"
+[[paths]]
```

Comment:
@pmac does this addition look okay to you? The work Alex did in #9735 ended up working & I want to make sure I don't mess it up with the CPG pages. :P

Reply:
I think it's not going to work because line 159 above this includes this path, so it will override this IIRC.

Reply:
So we'll have to specify a config for each subdomain of `mozorg/about` or something like that :(

Reply:
It seems like the only subdirectory in `mozorg/about` for fluent is `governance/policies/` could I replace `en/mozorg/about/**/*.ftl` with what I have?

Reply:
Hmm... I just fear that if/when we add more it will not behave properly unless the dev at that point remembers to update the config. Maybe we leave out the more broad config for now to be safe?

Reply:
I pushed a commit to delete the broader config. Is that what you meant?

Reply:
I think so. I guess it's safer to not include those potential new files in pontoon at all than too broadly. The other option I guess is to just have the subset of locales apply to everything in `mozorg/about` like you suggested above. I'm not sure which is best TBH. What do you think?

Reply:
I think this works for now. We can make this broader as we continue porting things over. It is hard to start broad in the beginning, imo.